src/server/models/classrooms.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+const mongoose = require("mongoose");
+const crypto = require("crypto");
+const util = require("../util");
+
+const classroomSchema = new mongoose.Schema({
+    id: { type: String, lowercase: true, unique: true, default: getId },
```

Comment:
You don't need to assign an ID manually. MongoDB automatically assigns every document an id in a field named `_id`. If this is meant to be something else, it should have a less confusing name.